src/modules/keyboardmanager/common/KeyboardManagerState.cpp

```diff
@@ -138,12 +157,24 @@ void KeyboardManagerState::UpdateDetectShortcutUI()
     }
 
     std::unique_lock<std::mutex> detectedShortcut_lock(detectedShortcut_mutex);
-    hstring shortcutString = convertVectorToHstring<DWORD>(detectedShortcut);
+
+    std::vector<hstring> shortcutKeys(detectedShortcut.size());
+    std::transform(
+        detectedShortcut.begin(), 
+        detectedShortcut.end(), 
+        shortcutKeys.begin(), 
+        [](DWORD k) { return winrt::to_hstring((unsigned)k); }
+    );

```

Comment:
I think some changes will be needed here when you pull the latest changes from the dev branch, since Shortcuts are refactored such that they are no longer a vector of keys, instead its a separate class. I think the class currently has a ToHstring which basically gives the combined space separated hstring as earlier. Based on that a ToVectorHstring can also be defined there which would return the `shortcutKeys` vector as required.

Reply:
Conflicts should be resolved now